web/src/components/planner/products/Product.vue

```diff
@@ -57,19 +57,12 @@
       </div>
       <div class="input-row d-flex align-center">
         <v-btn
-          v-if="!factory.parts[product.id]?.satisfied"
+          v-if="!factory.parts[product.id]?.satisfied || factory.dependencies.metrics[product.id]?.difference < 0"
           class="rounded mr-2"
           color="green"
           @click="fixProduction(factory, product.id)"
         ><i class="fas fa-wrench" /><span class="ml-1">Fix Production</span>
```

Comment:
Tbh I'd rename this from Fix Production to simply just "Fix" as it can come from multiple sources